src/Microsoft.Identity.Web/Constants/IDWebErrorMessage.cs

```diff
@@ -50,6 +50,7 @@ internal static class IDWebErrorMessage
         public const string ExceptionAcquiringTokenForConfidentialClient = "IDW10501: Exception acquiring token for a confidential client. ";
         public const string MicrosoftIdentityWebChallengeUserException = "IDW10502: An MsalUiRequiredException was thrown due to a challenge for the user. " +
            "See https://aka.ms/ms-id-web/ca_incremental-consent. ";
+        public const string ProvidedAuthenticationSchemeIsIncorrect = "IDW10503: Cannot determine the cloud Instance. The provided authentication scheme was {0}. Microsoft.Identity.Web inferred {1} as the authentication scheme. Available authentication schemes are {2}. See https://aka.ms/id-web/authSchemes. ";
```

Comment:
I'd propose we add quotes around the parameters. Sometimes the auth scheme is empty. This will make it more visible. For instance:

![image](https://user-images.githubusercontent.com/13203188/160502932-011acb7b-95d7-47dd-a351-b5b278b05f36.png)


```suggestion
        public const string ProvidedAuthenticationSchemeIsIncorrect = "IDW10503: Cannot determine the cloud Instance. The provided authentication scheme was '{0}'. Microsoft.Identity.Web inferred '{1}' as the authentication scheme. Available authentication schemes are '{2}'. See https://aka.ms/id-web/authSchemes. ";
```